bokeh/embed/bundle.py

```diff
@@ -223,13 +246,58 @@ def _bundle_extensions(objs, resources):
             continue
         names.add(name)
         module = __import__(name)
-        ext = ".min.js" if resources is None or resources.minified else ".js"
-        artifact = join(dirname(module.__file__), "dist", name + ext)
-        if exists(artifact):
-            bundle = BaseResources._inline(artifact)
-            extensions.append(bundle)
+        this_file = abspath(module.__file__)
+        base_dir = dirname(this_file)
+        dist_dir = join(base_dir, "dist")
+
+        ext_path = join(base_dir, "bokeh.ext.json")
+        if not exists(ext_path):
```

Comment:
This caused a major regression in Panel, we did not bundle the bokeh.ext.json or package.json until now. Should have reviewed this more closely.

Reply:
`package.json` is still optional, though useful to have if one wants to publish an extension to npm. However, `bokeh.ext.json` has been a part of the, let's call it, API since the new compiler was introduced, but wasn't used by embed code so far, which was a bug in itself. We need to set up a better testing and review process (you can see what the review focused on).